CHANGELOG.rst

```diff
@@ -7,6 +7,25 @@ The format is based on `Keep a Changelog`_,
 and this project adheres to `Semantic Versioning`_.
 
 
+Version 1.3.0 (2024-07-17)
+--------------------------
+
+* Added: ``strict`` argument
+  to ``audformat.utils.hash()``.
```

Comment:
This has a full stop at the end of the sentence. The others do not. What is convention that we are using here?

Reply:
The convention is to not use a full stop at the last sentence in an entry. But for earlier ones we use a full stops:
```
* Added: ``strict`` argument
  to ``audformat.utils.hash()``.
  If set to ``True``,
  the order of the data,
  and its level/column names
  are taken into account
  when calculating the hash
```
As you can see the last `hash` has no full stop.